cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -32,9 +34,9 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/yaml"
```

Comment:
I was a little concerned that k8s.io/apimachinery/pkg/util/yaml.Unmarshal might not be doing the same as sigs.k8s.io/yaml.Unmarshal (I vaguely remember some trouble like this with YAML or JSON Unmarshaller in the past) but after checking the code it looks like the former calls the latter, so should be ok